README.md

```diff
@@ -45,7 +45,7 @@ module.exports = {
         include: path.resolve("src"),
         use: [
           "thread-loader",
-          "expensive-loader"
+          // your expensive loader to thread
```

Comment:
`// your expensive loader (e.g babel-loader)` 

Or any existing loader you prefer (which is commonly used) :)